src/lib/autocomplete/public-api.ts

```diff
@@ -9,4 +9,4 @@
 export * from './autocomplete';
 export * from './autocomplete-module';
 export * from './autocomplete-trigger';
-
+export * from './autocomplete-origin';
```

Comment:
Should there be an update in the public API goldens? I'm afraid it's not failing at the moment.